nengo_spinnaker/builder/ensemble.py

```diff
@@ -34,6 +35,7 @@ def get_ensemble_sink(model, connection):
 
     if (isinstance(connection.pre_obj, nengo.Node) and
             not callable(connection.pre_obj.output) and
+            not isinstance(connection.pre_obj.output, Process) and
```

Comment:
I've added a test in `test/builder/test_ensemble.py` which tests just this line.  Other than that tests seem good, thank you.